assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -0,0 +1,199 @@
+  width: 20px;
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  margin-right: 7px;
+  border-radius: 100%;
+  color: ${({ theme }) => theme.colors.bg.panel};
+  background-color: ${({ theme, error }) =>
+    error ? theme.colors.fg.negative : theme.colors.fg.warning};
+`;
+
+const PanelTitle = styled.span`
+  color: ${({ theme, error }) =>
+    error ? theme.colors.fg.negative : theme.colors.fg.warning};
+`;
+
+const Row = styled.div`
+  margin-bottom: 25px;
+  padding-left: 12px;
+`;
+
+const PageIndicator = styled(Label)`
+  display: flex;
+  align-items: center;
+  margin-bottom: 12px;
+  svg {
+    height: 14px;
+    width: 14px;
+    margin-right: 6px;
+  }
+`;
+
+const ChecklistTab = (props) => {
+  const { checklist } = props;
+
+  const { highPriority, recommended, pages } = useMemo(
+    () =>
+      checklist.reduce(
+        (prevMessages, current) => {
+          const id = uuidv4();
+          const currentMessage = { ...current, id };
+          const isHighPriority =
+            currentMessage.type === PRE_PUBLISH_MESSAGE_TYPES.ERROR;
+
+          const [typeKey, copyKey] = isHighPriority
+            ? ['highPriority', 'recommended']
+            : ['recommended', 'highPriority'];
+
+          if (currentMessage.page !== undefined) {
+            const prevTypeGroup = prevMessages.pages[typeKey] || {};
+            const prevPageGroup = prevTypeGroup[currentMessage.page] || [];
+
+            let prevLengths = { highPriority: 0, recommended: 0 };
+            if (prevMessages.pages?.lengths) {
+              ({ lengths: prevLengths } = prevMessages.pages);
+            }
+
+            return {
+              recommended: prevMessages.recommended,
+              highPriority: prevMessages.highPriority,
+              pages: {
+                ...prevMessages.pages,
+                [typeKey]: {
+                  ...prevTypeGroup,
+                  [currentMessage.page]: [...prevPageGroup, currentMessage],
+                },
+                lengths: {
+                  ...prevLengths,
+                  [typeKey]: prevLengths[typeKey] + 1,
+                },
+              },
+            };
+          }
+
+          const groupedMessages = {
+            [copyKey]: prevMessages[copyKey],
+            [typeKey]: [...prevMessages[typeKey], currentMessage],
+          };
+          return { pages: prevMessages.pages, ...groupedMessages };
+        },
+        {
+          highPriority: [],
+          recommended: [],
+          pages: {},
+        }
+      ),
+    [checklist]
+  );
+
+  const renderRow = useCallback(({ message, id }) => {
+    return <Row key={`guidance-${id}`}>{message}</Row>;
+  }, []);
+
+  const renderPageGroupedRow = (entry) => {
+    const [pageNum, messages] = entry;
+    return (
+      <Fragment key={`guidance-page-group-${pageNum}`}>
+        <PageIndicator>
+          <Rectangle />
+          {sprintf(
+            /* translators: %s: the page number where the checklist issue is. */
```

Comment:
```suggestion
            /* translators: %s: page number. */
```

Let's use the same translators comment everywhere for this string.